sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -136,3 +136,20 @@ CREATE UNIQUE INDEX `uuid` ON `insurance_data` (`uuid`);
 ALTER TABLE `facility` ADD `weno_id` VARCHAR(10) DEFAULT NULL;
 #EndIf
 
+#IfNotRow background_services name X12_SFTP
+INSERT INTO `background_services` (`name`, `title`, `active`, `running`, `next_run`, `execute_interval`, `function`, `require_once`, `sort_order`) VALUES
+('X12_SFTP', 'SFTP Claims to X12 Partner Service', 1, 0, '2021-01-18 11:25:10', 1, 'start_X12_SFTP', '/library/billing_sftp_service.php', 100);
```

Comment:
since this is active, does this mean it will be running out of box?

Reply:
BG Services will call the function, but only perform the task if the global is enabled. There's no UI for enabling/disabling background services. It's pretty low-overhead to just call the function in billing_sftp_service.php

Reply:
If there is a global that turns this feature on, then would turn this on/off from globals when it is switched on/off. Can use the checkBackgroundServices() in edit globals script to (where can make a call to updateBackgroundService() for the service to turn it on/off).

Reply:
Cool!